src/java.base/share/classes/sun/net/www/MimeTable.java

```diff
@@ -198,42 +183,13 @@ public synchronized MimeEntry findByExt(String fileExtension) {
         return extensionMap.get(fileExtension);
     }
 
-    public synchronized MimeEntry findByDescription(String description) {
-        Enumeration<MimeEntry> e = elements();
-        while (e.hasMoreElements()) {
-            MimeEntry entry = e.nextElement();
-            if (description.equals(entry.getDescription())) {
-                return entry;
-            }
-        }
-
-        // We failed, now try treating description as type
-        return find(description);
-    }
-
-    String getTempFileTemplate() {
-        return tempFileTemplate;
-    }
-
     public synchronized Enumeration<MimeEntry> elements() {
         return entries.elements();
     }
 
-    // For backward compatibility -- mailcap format files
-    // This is not currently used, but may in the future when we add ability
-    // to read BOTH the properties format and the mailcap format.
-    protected static String[] mailcapLocations =
-            new String[]{
-                    System.getProperty("user.mailcap"),
-                    StaticProperty.userHome() + "/.mailcap",
-                    "/etc/mailcap",
-                    "/usr/etc/mailcap",
-                    "/usr/local/etc/mailcap"
-            };
-
```

Comment:
This field hasn't been used since 2007, the epoch of the Git history, it is probably predating even that. I doubt if it will ever be used or is still relevant. Hence, removed it.

Reply:
I don't think we have a deprecation process for this kind of dependency but dropping these deserves a CSR since the behavior is changing (and a release note).

Reply:
>behavior is changing

What behavior is changing?

Reply:
My mistake, that list of locations is really not being used. (And I agree it won't be coming back).